src/components/common/header/Header.js

```diff
@@ -1,29 +1,45 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { NavLink } from 'react-router-dom';
 import logo from '../../../../public/images/site-logo.png';
 import '../../../../public/js/NavBar';
 import Navbar from './NavBar';
 
-const Header = () => (
-  <header>
-    <nav className="navbar navbar-expand-lg">
-      <button className="navbar-toggler" type="button" data-toggle="offcanvas" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-        <i className="fas fa-bars" />
-      </button>
-      <button className="btn btn-link my-sm-0 d-inline d-sm-inline d-lg-none dropdown-toggle" type="submit" id="navbarSearchLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-        <i className="fas fa-search" />
-      </button>
-      <div className="dropdown-menu mobile-search-bar" aria-labelledby="navbarSearchLink">
-        <form className="form-inline my-lg-0">
-          <input className="form-control" type="search" placeholder="Search..." aria-label="Search" />
-        </form>
-      </div>
-      <NavLink to="/" className="navbar-brand">
-        <img src={logo} alt="logo" />
-      </NavLink>
-      <Navbar />
-    </nav>
-  </header>
-);
+class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.handleSearchSubmit = this.handleSearchSubmit.bind(this);
+  }
+
+  handleSearchSubmit(e) {
+    e.preventDefault();
+    const searchTerm = e.target.searchInput.value;
+    window.location = `/search?search=${searchTerm}`;
```

Comment:
Why are you using the `window.location` native property in your React component. React provides all this out of the box for you in `this.props`. Refer to this: [https://reacttraining.com/react-router/core/api/location](url)

Reply:
I will check this out